python/src/cerebellum/planners/anthropic.py

```diff
@@ -481,7 +495,7 @@ def format_into_messages(
         messages.extend([msg0, msg1])
 
         for past_step_idx, past_step in enumerate(session_history):
-            options = MsgOptions(mouse_position=False, screenshot=False, url=True)
+            options = MsgOptions(mouse_position=False, screenshot=False, tabs=False)
 
             if past_step_idx <= (len(session_history) - self.screenshot_history):
```

Comment:
it looks like `MsgOptions` no longer has a `url` attribute. I don't believe that attribute is expected anywhere so this line may be redundant.